src/components/LawProposals.vue

```diff
@@ -12,42 +13,65 @@ export default defineComponent({
 
   data() {
     return {
-      selectedLaw: undefined as LawId | undefined,
+      acceptedIds: [] as LawId[],
+      rejectedIds: [] as LawId[],
     }
   },
 
-  props: {
-    proposedLaws: Array as PropType<Law[]>,
-  },
+  computed: {
+    ...mapGetters(["proposedLaws"]),
 
-  methods: {
-    select(lawId: LawId) {
-      this.selectedLaw = lawId
+    removing() {
+      return (id: LawId) => this.acceptedIds.includes(id) || this.rejectedIds.includes(id)
     },
+  },
 
+  methods: {
     accept(lawId: LawId) {
-      this.store.dispatch("acceptLaw", { lawId })
+      this.acceptedIds.push(lawId)
     },
 
     decline(lawId: LawId) {
-      this.store.dispatch("rejectLaw", { lawId })
+      this.rejectedIds.push(lawId)
+    },
+
+    handle(lawId: LawId) {
+      this.removeAndDispatch(lawId, this.acceptedIds, "acceptLaw")
+      this.removeAndDispatch(lawId, this.rejectedIds, "rejectLaw")
+    },
+
+    removeAndDispatch(lawId: LawId, activeIds: LawId[], action: keyof Actions) {
+      const i = activeIds.indexOf(lawId)
+      if (i < 0) return
+      activeIds.splice(i, 1)
+      this.store.dispatch(action, { lawId })
     },
+
   },
 })
 </script>
 
 <template>
   <div class="ProposedLaws">
-    <div v-for="(law, index) in proposedLaws" :key="index" class="Law" :class="{ selected: law.id === selectedLaw }">
-      <div @click="select(law.id)">
-        <h3>{{ law.title }}</h3>
-        <div>{{ law.description }}</div>
-
-        <div class="button-bar">
-          <button class="accept" @click="accept(law.id)">✓</button>
-          <button class="decline" @click="decline(law.id)">✗</button>
+    <div
+      v-for="law in proposedLaws"
+      :key="law.id"
+      class="Law"
+      :class="{ removing: removing(law.id) }"
+      @animationend="handle(law.id)"
+    >
+      <input type="radio" name="selectedLaw" :id="law.id" />
+      <label :for="law.id">
+        <div>
```

Comment:
Wenn du das `<input>` in das `<label>` einbettest, brauchst du weder die `id` noch das `for`.

Reply:
Habe es probiert. Ging so spontan leider nicht.